### PR TITLE
avtart page in design chenges

### PR DESCRIPTION
**Pull request by archiswebiots**
avtart page in design chenges

### CODE REVIEW

File: assets/src/pug/pages/template/avatars.pug [hunk 1]
```diff
@@ -60,12 +60,13 @@ html(lang='en')
                       .avatars
                         div.avatar
                           img.img-100.rounded-circle(src='../assets/images/user/1.jpg', alt='#')
```
Comment (webiotsanjali) [inline assets/src/pug/pages/template/avatars.pug:62]: alt="#" do not put # in all alt.you need to put image name proper

File: assets/src/scss/components/_avatars.scss [hunk 2]
```diff
@@ -137,51 +137,27 @@
...
         animation: load 1.5s ease-out infinite;
-        position: absolute;
-        top: 9px;
-        right: 13px;
     }
     @keyframes load {
         0% {
-            background: transparent;
-            border: 3px solid transparent;
+            background-color: rgb(54, 155, 54);
```
Comment (webiotsanjali) [inline assets/src/scss/components/_avatars.scss:148]: use css variable only.not use color directly

Comment (archiswebiots) [reply assets/src/scss/components/_avatars.scss:148]: Screenshot from 2023-02-28 13-49-07

File: assets/src/pug/pages/template/avatars.pug (hunk 1 above)
Comment (archiswebiots) [reply assets/src/pug/pages/template/avatars.pug:62]: Screenshot from 2023-02-28 13-50-27

**Pull request merged**